.github/workflows/release.yml

```diff
@@ -0,0 +1,154 @@
+name: Release
+on:
+  push:
+    tags:
+      - "v*.*.*"
+env:
+  MACOSX_DEPLOYMENT_TARGET: 10.7
+
+jobs:
+  create-release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    outputs:
+      rlwy_version: ${{ env.CLI_VERSION }}
+
+    steps:
+      - name: Get the release version from the tag
+        shell: bash
+        if: env.CLI_VERSION == ''
+        run: |
+          # Apparently, this is the right way to get a tag name. Really?
```

Comment:
I ran into the same thing a while ago. Some data model stuff for GH Actions don't make sense.